packages/ui/src/components/activityBanner/ActivityBanner.component.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import * as Styled from './activityBanner.styled';
+
+const ActivityBanner = ({ backgroundImage, buttonImage, text, onClick }) => {
+  return (
+    <Styled.Banner $backgroundImage={backgroundImage}>
+      <span>{text}</span>
+      <Styled.ImgBtn alt="new organization" src={buttonImage} onClick={onClick} />
```

Comment:
The compiler was integrated with [SVGR](https://react-svgr.com/), so you may use SvgComponent directly.